### ISSUE TITLE
Create calculate_loss support

### ISSUE THREAD

**Issue by franchuterivera** [automl/Auto-PyTorch#97]
We want to make sure we have a minimization problem in all of our optimization contexts. We should follow autosklearn, in particular [here](https://github.com/automl/auto-sklearn/blob/598d33d0bc3438cdf3e7c1952f93afb6999943a0/autosklearn/metrics/__init__.py#L416).

This has to apply to individual models loss calculation (train evaluator, test evaluator) but also in the ensemble builder. Also calculate_score has to be update to return the real function value, not the result of the scorer, which is always a maximization problem.
